### PR TITLE
Add Advantech interfaces

### PR DESCRIPTION
**Pull request by jahuang0824**

### CODE REVIEW

File: dtmi/advantech/icam_500-1.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "@id": "dtmi:Advantech:ICAM_500;1",
+  "@type": "Interface",
+  "displayName": "ICAM-500",
+  "contents": [
+    {
+      "@type": "Component",
+      "name": "deviceinfo",
```
Comment (digimaun) [inline dtmi/advantech/icam_500-1.json:8]: deviceinfo here doesn't have consistent character casing compared to other usage (and our best practice guidelines) i.e. `deviceinfo` -> `deviceInfo`

File: dtmi/advantech/icam_500-1.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      "schema": "dtmi:Advantech:JetsonDeviceInformation;1"
+    },
+    {
+      "@type": "Component",
+      "name": "thermalzones",
```
Comment (digimaun) [inline dtmi/advantech/icam_500-1.json:13]: Should the name of this component be `thermalZones` or `Thermal_Zones`?

File: dtmi/advantech/icam_500-1.json (hunk 1 above)
Comment (jahuang0824) [reply dtmi/advantech/icam_500-1.json]: updated as suggestion

File: dtmi/advantech/icam_500-1.json (hunk 2 above)
Comment (jahuang0824) [reply dtmi/advantech/icam_500-1.json]: updated as suggestion

**Comment by digimaun** [comment]
LGTM

**Review by digimaun — APPROVED** [review]

**Pull request merged**